src/Routers/SchemasRouter.js

```diff
@@ -14,16 +14,34 @@ function classNameMismatchResponse(bodyClass, pathClass) {
   );
 }
 
+function injectDefaultSchema(schema) {
+  if (Array.isArray(schema)) {
+    let schemas = schema.map((s) => {
```

Comment:
Rather than have a function that accepts a schema or an array of schemas, can we have one that only accepts a single schema, and do the `map` outside the function if it needs it?